org.OpenT2T.Sample.SuperPopular.Smoke/Wink Alarm/js/manifest.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest>
```

Comment:
The folder name should also be updated to Wink Smoke Detector, I think. Wink in their API docs call it Smoke Alarm so that would be acceptable as well.